.github/workflows/expo_cd.yml

```
@@ -1,26 +1,93 @@
+          NEW_VERSION_NAME="$MAJOR.$MINOR.$PATCH"
+          
+          sed -i "s/VERSION_CODE=.*/VERSION_CODE=$NEW_VERSION_CODE/" version.properties
+          sed -i "s/VERSION_NAME=.*/VERSION_NAME=$NEW_VERSION_NAME/" version.properties
+          
+          echo "NEW_VERSION_CODE=$NEW_VERSION_CODE" >> $GITHUB_ENV
+          echo "NEW_VERSION_NAME=$NEW_VERSION_NAME" >> $GITHUB_ENV
+          echo "BUMPED=true" >> $GITHUB_ENV
+
+      - name: Set current version for notification
+        if: ${{ github.event.inputs.version_bump_type == 'None' }}
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:3:42: Double quote to prevent globbing and word splitting

(shellcheck)